examples/poem_example/core/src/mutation.rs

```diff
@@ -5,7 +5,7 @@ pub struct Mutation;
 
 impl Mutation {
     pub async fn create_post(
-        db: &DbConn,
+        db: &DatabaseTransaction,
         form_data: post::Model,
     ) -> Result<post::ActiveModel, DbErr> {
```

Comment:
Then, change the `db` to takes any types implemented `ConnectionTrait`:
```rs
pub async fn create_post<C: ConnectionTrait>(
    db: &C,
    form_data: post::Model,
)
```

Reply:
Would be an idea, yes. But I wanted to show, that you can use transactions just like the normal connections.